src/main/java/org/apache/pirk/querier/wideskies/encrypt/EncryptQuery.java

```diff
@@ -156,10 +149,23 @@ public Querier encrypt(int numThreads) throws InterruptedException, PIRException
     return new Querier(selectors, paillier, query, embedSelectorMap);
   }
 
+  /**
+   * Use this method to get a securely generated, random string of 2*numBytes length
+   *
+   * @param numBytes How many bytes of random data to return.
+   * @return Random hex string of 2*numBytes length
+   */
+  private String getRandByteString(int numBytes)
```

Comment:
I'm going to leave it in EncryptQuery because no other class should ever generate a new Hash Key.